tests/selenium/test_selenium_chat_flow.py

```
@@ -369,3 +369,238 @@ def test_explain_denial_flow_end_to_end(self):
+        # (a new one will be assigned when user sends a message)
+        assert new_chat_id is None or new_chat_id != initial_chat_id, \
+            "New Chat should clear or change the chat ID"
+
+        # User info should still be preserved
+        user_info_json = self.execute_script(
+            "return localStorage.getItem('fhi_user_info');"
+        )
+        assert user_info_json is not None
+
+        print("✓ New Chat button works for existing users")
+        print(f"  - Old chat ID cleared")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)